### ISSUE TITLE
Ground Glass Opacities

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7113]
**Preferred term label:**
Ground Glass Opacities

**Synonyms**
ground-glass opacification
GGO

**Definition (free text, please give PubMed ID)**
Focal or diffuse areas of ill-defined, hazy, increased lung attenuation, which cause pulmonary vascular indistinctness, yet through which vessels can still be identified via high-resolution computed tomography or the conventional radiograph. [PMID: 12465687]

**Parent term (use hpo.jax.org/app)**
~HP:0002088 - Abnormal lung morphology~
HP:0031983 - Abnormal pulmonary thoracic imaging finding _(updated 2021-09-13)_

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
[DCM:112120](http://dicom.nema.org/medical/Dicom/2016b/output/chtml/part16/chapter_D.html#DCM_112120)
SNOWMEDCT_US63531000122103 [(emerging concept - not yet released)](https://www.mscbs.gob.es/profesionales/hcdsns/areaRecursosSem/snomed-ct/SNOMED_doc/Conceptos_relacionados_SARS-CoV-2-Version11.0_20201130_en.pdf)

**Comment by pnrobinson** [comment]
This term already exists
Ground-glass opacification HP:0025179
https://hpo.jax.org/app/browse/term/HP:0025179

I will add the above label as a synonym, thanks!
